hooks/beforeRegistration.ts

```diff
@@ -1,35 +1,16 @@
 import { currentStoreView } from '@vue-storefront/core/lib/multistore'
 
 export function beforeRegistration({ Vue, config, store, isServer }) {
-  const VSF_PAYPAL_CODE = 'vsfpaypal'
-  store.dispatch('payment/addMethod', {
-    'title': 'Paypal',
-    'code': VSF_PAYPAL_CODE,
-    'cost': 0,
-    'costInclTax': 0,
-    'default': false,
-    'offline': true
-  })
+  // const VSF_PAYPAL_CODE = 'vsfpaypal'
```

Comment:
@tonyisworking We can remove the unnecessary comments.
Thanks! 
I will push an improvements to this branch, if you will allow

Reply:
> I will push an improvements to this branch, if you will allow

@dimasch Yes you can do that.

Reply:
Do i need to anything for that?